### DISCUSSION TITLE
tr/datasets/obb/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#16929] [Docs]
# tr/datasets/obb/

Ultralytics YOLO  modelleri için OBB veri kümesi formatlarını keşfedin. Nesne algılama eğitiminizi geliştirmek için yapıları, uygulamaları ve format dönüşümleri hakkında bilgi edinin.

https://docs.ultralytics.com/tr/datasets/obb/

**Comment by enescanerkan** [comment]
from ultralytics.data.converter import convert_dota_to_yolo_obb

convert_dota_to_yolo_obb("path/to/DOTA")

This code only starts converting the DOTA format dataset to YOLO-OBB format. What I am looking for is to convert my YOLO format labels to YOLO-OBB format. In your documentation, you show that you converted the labels you gave in yolo format to yolo obb format, but this code does not help in any way. 

For example, can't I convert my yolo format label from class_index x1 y1 x2 y2 to → class_index x1 y1 x2 y2 x3 y3 x4 y4? Isn't the process we call oriented bounding box converting from the format I wrote above to obb format?

**Reply by glenn-jocher** [reply]
@enescanerkan to convert YOLO format labels to YOLO-OBB format, you'll need to manually adjust your labels to include the four corner points required for OBB. Unfortunately, there isn't a direct conversion tool for this specific task in the Ultralytics library. For more details on the YOLO-OBB format, please visit our documentation.
